mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala

```diff
@@ -22,7 +22,13 @@ trait ArgCompletions { this: MetalsGlobal =>
     val method: Tree = typedTreeAt(funPos)
     val methodSym = method.symbol
     lazy val baseParams: List[Symbol] =
-      if (method.tpe == null) {
+      if (methodSym.isModule) {
```

Comment:
My understanding of this code is as follows:
if it's an object take the companion class and get args for its constructor.
What if we want arg completions for an `apply` defined in the companion object? Will that work?
```
case class A(val a: Int, val b: Int)
object A {
  def apply(a: Int): A = new A(a, 3)
}
```

Reply:
Thanks for noticing. I changed it so instead of using companion class, we search for `apply` method and we prefer the one defined in companion object.